### PR TITLE
Feat/mobile improvements

### LINKED ISSUE #158: Mobile UI/UX improvements
The mobile experience has room for improvements. @andreforsousa will comment below on that.

- [x] Reduce copy lines in hero
- [x] "Cards" approach on projects and videos carousel
- [x] Community CTAs should be 100%

### PR DESCRIPTION
**Pull request by PedroMiguelSS**
This PR implements a few mobile improvements such as:
1. reduced copy lines on `hero` section;
2. redesign of `What people are building with it` carousel;
3. redesign of `Publications & Talks` carousel;
4. bigger CTAs on `Community` section.

It closes #158

### CODE REVIEW

File: src/shared/components/carousel/index.css [hunk 1]
```diff
@@ -289,18 +288,34 @@
   opacity: 0.25;
   color: black;
   font-family: "slick";
-  font-size: 6px;
+  font-size: 24px;
   font-smoothing: antialiased;
   text-align: center;
   line-height: 20px;
+  transition:
+    color 300ms ease,
```
Comment (fsdiogo) [inline src/shared/components/carousel/index.css]: Variable for the `300ms`.

File: src/shared/components/carousel/index.js [hunk 2]
```diff
@@ -16,53 +16,74 @@ const settings = {
...
 }
 
-const Carousel = ({ itemsList, modifier, size, onVideoClick, activeIndex, translationsList, isMobile }) => {
-  let items
-  const numberOfSlidesToShow = isMobile ? 1 : size
-  const finalSettings = { ...settings, slidesToShow: numberOfSlidesToShow }
+class Carousel extends Component {
+  render () {
+    const { itemsList, modifier, size, onVideoClick, activeIndex, translationsList, isMobile } = this.props
+    let items
```
Comment (fsdiogo) [inline src/shared/components/carousel/index.js]: Put all the `const` first, then the `let`. Sorry for the nitpick, it's my OCD hehe.

File: src/shared/components/carousel/index.css [hunk 3]
```diff
@@ -289,18 +288,34 @@
...
+    font-size 300ms ease,
+    opacity 300ms ease;
```
Comment (vasco-santos) [inline src/shared/components/carousel/index.css]: Can you add a variable for the above transition times?

File: src/shared/components/hero-section/index.js [hunk 4]
```diff
@@ -99,10 +104,18 @@ class Hero extends Component {
       { count: formattedDownloads }
     )
 
-    const info = {
-      currentVersionStr,
-      latestUpdateDateStr,
-      downloadsStr
+    if (isMobile) {
```
Comment (fsdiogo) [inline src/shared/components/hero-section/index.js]: If you want to save lines (optional):

```js
const info = isMobile ?
  { `${currentVersionStr} | ${latestUpdateDateStr}`, downloadsStr } :
  { currentVersionStr, latestUpdateDateStr, downloadsStr }
```

File: src/shared/components/carousel/index.js [hunk 5]
```diff
@@ -16,53 +16,74 @@ const settings = {
...
+      centerPadding: '12%'
+    }
+  }]
 }
 
-const Carousel = ({ itemsList, modifier, size, onVideoClick, activeIndex, translationsList, isMobile }) => {
-  let items
-  const numberOfSlidesToShow = isMobile ? 1 : size
-  const finalSettings = { ...settings, slidesToShow: numberOfSlidesToShow }
+class Carousel extends Component {
```
Comment (vasco-santos) [inline src/shared/components/carousel/index.js]: Why did you change this to a non functional component?

File: src/shared/components/youtube/index.js [hunk 6]
```diff
@@ -1,43 +1,86 @@
...
+import React, { PureComponent } from 'react'
 import classNames from 'classnames'
 import Svg from 'shared/components/svg'
 import styles from './index.module.css'
 import playSvg from './media/play.sprite.svg'
+import ReactYoutube from 'react-youtube'
 
-class Youtube extends Component {
-  state = { clicked: false }
+const opts = {
```
Comment (fsdiogo) [inline src/shared/components/youtube/index.js]: Why is this outside the component scope?

File: src/shared/components/youtube/index.module.css [hunk 7]
```diff
@@ -9,6 +9,7 @@
   overflow: hidden;
   padding-bottom: 56.23%;
   background: #000;
+  border-radius: 5px;
```
Comment (fsdiogo) [inline src/shared/components/youtube/index.module.css]: Variable for this, as you're using it tons of times in this file!

File: src/shared/components/hero-section/index.js (hunk 4 above)
Comment (PedroMiguelSS) [reply src/shared/components/hero-section/index.js]: I think you missed `versionAndDateStr:`
It should be: 
```js
const info = isMobile
      ? { versionAndDateStr: `${currentVersionStr} | ${latestUpdateDateStr}`, downloadsStr }
      : { currentVersionStr, latestUpdateDateStr, downloadsStr }
```
But I got it. Thank you for your suggestion!

File: src/shared/components/youtube/index.js (hunk 6 above)
Comment (PedroMiguelSS) [reply src/shared/components/youtube/index.js]: It shouldn't. Thanks.

File: src/shared/components/carousel/index.js (hunk 5 above)
Comment (PedroMiguelSS) [reply src/shared/components/carousel/index.js]: I changed it to a non functional component because I was using an approach that requires this component t be a non-functional component. However, I did it other differently and I forgot to restore this component back to a functional component. 
Thank you!
